### DISCUSSION TITLE
Graphical Output

### DISCUSSION THREAD

**Discussion by Zach-Harald** [amidaware/tacticalrmm#1405] [Q&A]
Hallo,

how can i do that the results of a script (Windows, Powershell) are displayed in a graphical box like the "CPU Load Check" (Show Run History)?

Thanks for help

**Comment by silversword411** [comment]
Huh?

**Reply by Zach-Harald** [reply]
Hi

**Reply by wh1te909** [reply]
what exactly are you asking? script check results already show in the the graph when you click on "Show Run History"

**Reply by Zach-Harald** [reply]
I want to write a script, that checks the number of pages which were printed on a pc per day. The script runs every day. How can i enable a graphical box, which shows me a line of the printed pages along the time?

**Reply by wh1te909** [reply]
the graph is only for pass/fail. you would need to program something yourself if you want a custom graph. you can see the current implementation [here](https://github.com/amidaware/tacticalrmm-web/blob/develop/src/components/graphs/CheckGraph.vue)

**Reply by Zach-Harald** [reply]
What kind of script is this? Where must I place it?

**Reply by wh1te909** [reply]
this is not a script. you would need to program it in javascript/vuejs and integrate it into the software. what you are asking is not possible unless you build your own solution or create an integration into our software

**Reply by Zach-Harald** [reply]
Thanks
